xds/internal/balancer/cdsbalancer/cdsbalancer.go

```diff
@@ -361,6 +363,14 @@ func (b *cdsBalancer) run() {
 					break
 				}
 				b.childLB.UpdateSubConnState(update.subConn, update.state)
+			case exitIdle:
+				if b.childLB == nil {
+					b.logger.Errorf("xds: received ExitIdle with no child balancer")
+					break
+				}
+				if ei, ok := b.childLB.(balancer.ExitIdle); ok {
+					ei.ExitIdle()
+				}
```

Comment:
Add a note that this assumes the child supports `ExitIdle()`.

Reply:
Done